### PR TITLE
In preprocess_files function check that the py file exists.

### PR DESCRIPTION
**Pull request by RotemAmit**
## Related Issues
fixes:

## Description: 
When running the pre-commit command with only a yml file as input, we want to collect the python file as well. Before we do that we want to make sure that it exists.

### CODE REVIEW

File: demisto_sdk/commands/pre_commit/pre_commit_command.py [hunk 1]
```diff
@@ -441,10 +441,11 @@ def preprocess_files(
             files_to_run.update({path for path in file.rglob("*") if path.is_file()})
         else:
             files_to_run.add(file)
-            # if the current file is a yml file, add the matching python file to files_to_run
+            # If the current file is a yml file, add the matching python file to files_to_run
             if str(file).endswith("yml"):
```
Comment (ilaner) [inline demisto_sdk/commands/pre_commit/pre_commit_command.py:445]:
```suggestion
            if file.suffix == ".yml"
```

**Comment by GuyAfik** [comment]
@RotemAmit no need for changelog here?

**Comment by eyalpalo** [comment]
@GuyAfik This is a fix to an already document change Rotem did, it is in the same release, so I think we don't need a new entry in the changelog

**Comment by RotemAmit** [comment]
@GuyAfik This is the related [PR](https://github.com/demisto/demisto-sdk/pull/3744).

**Pull request merged**